### PR TITLE
build 3.15.0

### PR DESCRIPTION
**Pull request by boldorider4**
- [Jira ticket](https://anaconda.atlassian.net/browse/PKG-3256)
- [Upstream project](https://github.com/pytest-dev/pytest-randomly)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - wheel
   run:
-    - python >=3.6
+    - python
     - importlib-metadata >=3.6.0
```
Comment (cbouss) [inline recipe/meta.yaml:25]:
```suggestion
    - importlib-metadata >=3.6.0 # [py<310]
```
https://github.com/pytest-dev/pytest-randomly/blob/3.15.0/setup.cfg#L36

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
+    - wheel
```
Comment (cbouss) [inline recipe/meta.yaml:22]:
```suggestion
    - wheel
    - setuptools
```

File: recipe/meta.yaml (hunk 1 above)
Comment (boldorider4) [reply recipe/meta.yaml:25]: Thank you! I missed that.

File: recipe/meta.yaml (hunk 2 above)
Comment (boldorider4) [reply recipe/meta.yaml:22]: thanks!

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d60c2db71ac319aee0fc6c4110a7597d611a8b94a5590918bfa8583f00caccb2
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name_url }}/{{ name_url }}-{{ version }}.tar.gz
+  sha256: b908529648667ba5e54723088edd6f82252f540cc340d748d1fa985539687047
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:16]: We should add `skip: true. # [py<38]` since that's what they officially support.

**Review by cbouss — APPROVED** [review]

**Pull request merged**